pkg/syncer/cnsoperator/controller/cnsregistervolume/util.go

```diff
@@ -152,7 +152,16 @@ func constructCreateSpecForInstance(r *ReconcileCnsRegisterVolume,
 	if instance.Spec.AccessMode == v1.ReadWriteOnce || instance.Spec.AccessMode == "" {
```

Comment:
remove 152:164

and directly set `createSpec.VolumeType = common.BlockVolumeType` since we don't support registering file volume.